.goreleaser.yml

```diff
@@ -58,6 +58,10 @@ before:
       sh -ec 'if [ {{ .IsSnapshot }} == false ] ; then
         make --no-print-directory CAREN_VERSION=v{{ trimprefix .Version "v" }} list-images >caren-images.txt
       fi'
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == false ] ; then
+        make --no-print-directory CAREN_VERSION=v{{ trimprefix .Version "v" }} add-version-to-clusterclasses
```

Comment:
Just curious, does `.IsSnapshot==false` when running e2e tests?

Reply:
No, that runs with `release-snapshot` so this will not be run. Do you think there's a problem with that?

Reply:
We may need this in the future when running upgrade e2e tests, but probably premature to worry about it now.

Reply:
I'm confused by your reply, because as I read it, it says "this code does not run when we use `release-snapshot`", but then what is the purpose of the conditional?

Reply:
`the conditional evaluates to false, so the branch does not run` is correct.